picodom.d.ts

```diff
@@ -1,26 +1,41 @@
-export interface VNode<Data> {
+export as namespace picodom
+
+export interface VirtualNode<Data> {
   tag: string
-  data?: Data
+  data: Data
   children: VElement<{}>[]
 }
 
 export interface Component<Data> {
-  (data?: Data, ...children: VElement<{}>[]): VNode<Data>
+  (data: Data, children: VElement<{}>[]): VirtualNode<any>
```

Comment:
Per [line 29 in h.js](https://github.com/picodom/picodom/blob/901a0c8a403244cfc8ec0d143aca036b208ba491/src/h.js#L29), children is an array of VirtualNodes or strings. Each child is not its own argument to the Component function